tests/test_views_permissions.py

```diff
@@ -61,6 +61,13 @@ def test_bucket_create_permission_exists(self):
             'resource_name': 'bucket',
             'uri': '/buckets'})
 
+    def test_can_paginate(self):
+        """kinto_http.js uses this sort of request when listing permissions"""
+        resp = self.app.get('/permissions?_sort=id&_limit=1', headers=self.everyone_headers)
+        while 'Next-Page' in resp.headers:
+            next_url = resp.headers['Next-Page'][len('http://localhost/v1'):]
+            resp = self.app.get(next_url, headers=self.everyone_headers)
```

Comment:
Ideally, I'd like the `/buckets` perm to come out first, could you please check that? (I never know whether `None` comes first or not)

Reply:
Apparently it always came last (per the test just above). `None` comes first, but `Missing` comes last to mimic the Postgres backend.